Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'config'
 # (pinning to < 4.1 until issue w/ webshims is resolved https://github.com/aFarkas/webshim/issues/560)
 gem 'jquery-rails', '< 4.1'
 # Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
-gem 'turbolinks'
+gem 'turbolinks', '~> 5.0'
```

Comment:
I don't even know what I comment like that might say.  TBH, this will implicit when we move to rails 5 anyway, so I'm not sure it's necessary to leave a breadcrumb at this point.